test.py

```diff
@@ -680,14 +680,14 @@ def set_summary(summary):
                         raise
                 set_summary("failed: {}".format(e))
             finally:
-                if self.is_new or self.is_equal_result is False:
-                    # Put a copy of the .reject file close to the .result file
-                    # so that it's easy to analyze the diff or overwrite .result
-                    # with .reject. Preserve the original .reject file: in
-                    # multiple modes the copy .reject file may be overwritten.
-                    shutil.copyfile(self.tmpfile, self.reject)
-                elif os.path.exists(self.tmpfile):
-                    pathlib.Path(self.tmpfile).unlink()
+                if os.path.exists(self.tmpfile):
+                    if self.is_executed_ok and (self.is_new or self.is_equal_result is False):
+                        # Move the .reject file close to the .result file
```

Comment:
The commit message now mentions the change from copy to move - but it only states the fact that the change happens.
However, in my previous review I asked for a **motivation**: explanation **why** the change was done (not the mere mentioning of the fact that it was done).

Right now the commit message only explains why we need the check for file existing (the first paragraph explains the reason). It does not explain why we want to move instead of copy.

I don't think there's any value in changing this PR yet another time, but for the future - please focus on explaining why you do the changes you do in the commit messages. If it's hard to motivate the change - maybe it shouldn't be done at all.